packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.service.js

```diff
@@ -0,0 +1,14 @@
+export default class PackMigrationMeetingService {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  searchMeetings(eligibilityReference, productCode, installationType) {
+    return this.$http.post(`/connectivity/eligibility/search/meetings`, {
+      eligibilityReference,
+      productCode,
+      installationType,
+    });
```

Comment:
```suggestion
  searchMeetings(eligibilityReference, productCode, installationType) {
    return this.$http.post(`/connectivity/eligibility/search/meetings`, {
      eligibilityReference,
      productCode,
      installationType,
    }).then(({data}) => data);
```
It's more easy for use if you return directly the datas